docker/environment.yml

```diff
@@ -5,8 +5,9 @@ channels:
 - conda-forge
 dependencies:
 - python>=3.6
+- pip
 - matplotlib==2.1.2
 - numpy==1.16.1
 - pip:
-  - -r REQUIREMENTS-STRICT.txt
+  - -r ../REQUIREMENTS-STRICT.txt
```

Comment:
it's not weird / broken to reach up to a parent directory?

Reply:
I agree it's slightly weird, yeah, but since the docker directory is effectively self-contained, the impact is pretty low, and it keeps the environment.yml file usable outside of docker.